examples/visualization_demo/Main.cc

```diff
@@ -162,6 +162,36 @@ void buildScene(ScenePtr _scene)
   comVisual->SetInertial(comVisualInertial);
   box->AddChild(comVisual);
 
+  // create joint child visual
+  VisualPtr jointChildBox = _scene->CreateVisual("joint_child");
+  jointChildBox->AddGeometry(_scene->CreateBox());
+  jointChildBox->SetOrigin(0.0, 0.0, 0.0);
+  jointChildBox->SetLocalPosition(3.5, 0.5, 0.0);
+  jointChildBox->SetLocalRotation(0, 0, 0);
+  jointChildBox->SetMaterial(blue);
+  root->AddChild(jointChildBox);
+
+  // create joint parent visual
+  VisualPtr jointParentBox = _scene->CreateVisual("joint_parent");
+  jointParentBox->AddGeometry(_scene->CreateBox());
+  jointParentBox->SetOrigin(0.0, 0.0, 0.0);
+  jointParentBox->SetLocalPosition(2.0, 0.5, 0.0);
+  jointParentBox->SetLocalRotation(0, 0, 0);
```

Comment:
use a different rotation from child to show what the parent axis looks like with rotation?

Reply:
[`9edace1`](https://github.com/ignitionrobotics/ign-rendering/pull/366/commits/9edace1dc0e8b9ab5ea2c86f1784db34472b267b)